### ISSUE TITLE
List of MODDB/ZDFORUMS issues after 31-10-21 release

### ISSUE THREAD

**Issue by Ozymandias81** [Drugod/BloomProject#2]
MODDB
:x:  When I shot Caleb plasmagun in water or underwater i got bugs with graphic. I have many flashing colors. They can cause epilepsy
> WONTFIX, USER'S PROBLEM --ozy81
- [x] BLM03: I can't seem to be able access the secret rocket box (Thing #467). Checked in GZDB and the "door" sector doesn't even have a tag (neither do its lines, double checked just in case)
- [x] BLM03: Also, playing as Caleb, picking up the Life Leech didn't trigger the secret.
- [x] BLM05: Can't get kitchen's secret, player won't fit into the sector
- [x] BLM05: The train's furnace is marked as secret, but has no goodies
- [x] BLM07: Is the secret Beast vision powerup reachable? Found no way to open the bars blocking it, not even in GZDB
- [x] BLM08: Penultimate boss spawn is borked. Idk what is supposed to spawn. I've gotten the boss with two companion hands, boss and 1 hand, boss only, and no boss but the two hands
- [x] HUD: Bullet Box HUD message on pickup is "BBOX"
- [x] HUD: Secret level rune HUD message on pickup is "Diving Suit"
- [x] When a Possesed zombieman is killed under water, the hand flying off his head does not die (and continuously babbles) until the player crushes it
- [x] Arachnopods of any kind often start jumping up and down incessantly. And already can not move normally
- [ ] Proximity dynamite works even on enemies standing behind obstacles (for example, a door), of course, without causing damage to them. Which makes it very difficult to set traps with this type of weapon
- [x] When picking up Proximity Dynamite, an item (in the native interface that does not have an icon) appears in the inventory, which allows you to take it in your hand. I'm not sure if that's what it was meant to be.

ZDFORUMS
- [x] on the second level after interacting with yellow door that triggers an explosion, you can use it again to make more explosions and block the path
- [x] on the third level(siege on pit) doors near the room with the word sloth on the wall are usable only once, the problem is only one of them is used to open both so you can block the path with using the other door
- [ ] setting status bar to 3 in the scaling options makes the hud cut off at the edge of the screen, setting to 2 makes it a bit too small
- [x] i dont think this is intentional but altfire for bfg uses all ammo

CAPTAIN J PLAYTESTING
- [x] Quite frankly Caleb's Single Barrel Shotgun acts just like the Sawed-Off but much, much slower. Colt has own purpose of being nicely accurate and i thought it would be the same, but weirdly it isn't. Also for some reason it plays no reloading sound.
- [ ] The Eyeball Cursor is quite awkward to use. I mean, where's the tip? When i click, it selects the wrong thing.
- [ ] Voodoo Doll takes absolute zero effect on Zombies coming out of their graves, ground and whatev
- [x] Severed Hand came off from Zombie enemy spaz out with the sound and never dies underwater unless you step on it. 
- [x] Apocalyptic_NeoCvltist and PossessedZombieMan play firing sound before they fire their gun.
- [ ] I think the thrown default TNT explodes rather too quickly if it hits the enemy. And it even shows vanilla doom explosion.
- [x] I just noticed that Ice_ArachnoPod and DeepOneTri drop TWO Voodoo Dolls. Intentional? Magma_ArachnoPod Does the same thing with Spray Can.
- [x] Rats are somewhat harder to hit, has extra health and even immune to Life Leech? I'm positive they die in one shot in Original Blood though...
- [x] You can use Tesla Cannon underwater if you have Akimbo Powerup.
- [x] Remote and Proxy Bomb disappears when they land on watery terrain. Not on underwater, but the flat floor.
- [x] Magma_ArachnoPod's corpse sprite seems stopped in the middle of the animation: http://i.imgur.com/IPKeX1t.png

BLM2:

- [ ] Dunno what's triggering it, but Ghost_Revenant explodes this particular wall and the debris of it has missing texture: http://i.imgur.com/IPKeX1t.png

BLM5:

- [x] The camera of this particular 3D skybox seems too close to the floor or something? http://i.imgur.com/NkUmqbJ.png
- [x] Enemies do fall into the Insta-Kill void but it's a weird sight that they don't die but keep attacking me. Like this one for example. I can't even shoot at it because there's some kind of weird floor that blocks my shot: http://i.imgur.com/V0pH9Je.png
- [x] If you break the sequence, like going in backwards, this platform doesn't go up and will softlock the game: http://i.imgur.com/ij5YQTJ.png
- [x] So i failed at riding onto the next train after overheating the engine. The screen goes white and doesn't kill me for some reason. Oh, also the text still says jump on the train and press the eye, even though you beat the level by just jumping onto it.

BLM6:

- [x] There's this invisible gap near the Hospital door: http://i.imgur.com/1uxvi4r.png

BLM7:

- [x] This tree does some good jigglin: http://i.imgur.com/Brh4K0w.png
- [x] That'd be "Skull Key Needed." http://i.imgur.com/09AKrZ8.png
- [x] Are these spinning door suppose to kill you or not? Because i do get killed by these doors Happy Tree Friends style: http://i.imgur.com/PRn4kiM.png
- [ ] That big clown boss guy is straight unfair because he fires explosives fast and how narrow the Carnival area is, especially the first place you enter. If he arrives at the exit, then it's game over.
- [x] Jojo plays no firing sound effect.
- [ ] This guy gets stuck all over the place: http://i.imgur.com/y4QSS4E.png
- [ ] Hey uhh i would like to talk about the big cerberus mancubus hybrid bosses... Why do they have Fireballs with Nuke level of splash damage? Since i have to fight so many of them this is plain overkill imo.
- [ ] And hey, the underwater mancubus bosses move fast underwater, does bullethell pattern and HAS Nuke Fireballs. Why does this not surprise me?
- [ ] Can i expect some kind of hint that i actually can lower the walls in final palace area where you have to fight 3 more bosses? http://i.imgur.com/hJ6yHKW.png
- [ ] Innocents bleed like freaking hell if they get hit by pinky beberus' fireball

**Comment by Drugod** [comment]
Blm05 related issues:

"f you break the sequence, like going in backwards, this platform doesn't go up and will softlock the game: 

 So i failed at riding onto the next train after overheating the engine. The screen goes white and doesn't kill me for some reason. Oh, also the text still says jump on the train and press the eye, even though you beat the level by just jumping onto it."

Doesnt find how to reproduce it, i have test it deeply i didnt find it, so i'll close it

**Comment by Drugod** [comment]
" Are these spinning door suppose to kill you or not? Because i do get killed by these doors Happy Tree Friends style:"

Yes should kill you as blood game do

**Comment by Drugod** [comment]
"Quite frankly Caleb's Single Barrel Shotgun acts just like the Sawed-Off but much, much slower. Colt has own purpose of being nicely accurate and i thought it would be the same, but weirdly it isn't. Also for some reason it plays no reloading sound." Fixed, but still doesnt sound the reload sound, idk the reason should take a look closely

**Comment by Drugod** [comment]
" Apocalyptic_NeoCvltist and PossessedZombieMan play firing sound before they fire their gun." Thats not correct, works fine
